service/service_test.go

```diff
@@ -11,13 +11,12 @@ import (
 
 	"github.com/golang/mock/gomock"
 	"github.com/spacemeshos/go-scale"
-	"github.com/spacemeshos/merkle-tree"
 	"github.com/stretchr/testify/require"
 	"golang.org/x/sync/errgroup"
 
 	"github.com/spacemeshos/poet/gateway/challenge_verifier"
 	"github.com/spacemeshos/poet/gateway/challenge_verifier/mocks"
-	"github.com/spacemeshos/poet/prover"
+	"github.com/spacemeshos/poet/service"
 )
 
 type MockBroadcaster struct {
```

Comment:
Can we use `gomock` for this?

Reply:
Broadcaster is going to be removed in #168